docs/src/api.md

```diff
@@ -100,6 +100,64 @@ ADC
 Delay
 ```
 
+## Functions for structs
```

Comment:
This is only for Sequence Events.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/258/commits/109eb655328e48204bc025bc331484464822a94a. Renamed to "Sequence-related Functions".